src/OpenSage.Game.Tests/InstalledFilesTestData.cs

```diff
@@ -7,15 +7,18 @@ namespace OpenSage.Data.Tests
 {
     internal static class InstalledFilesTestData
     {
+        private static readonly IInstallationLocator locator;
```

Comment:
It's a minor thing, but please could you follow the naming convention I've used in the rest of the code, and name this `Locator`.